ocs_ci/deployment/hosted_cluster.py

```diff
@@ -913,7 +913,7 @@ def _apply_storage_client_cr():
 
             storage_client_data["spec"]["onboardingTicket"] = onboarding_key
 
-            self.storage_client_name = storage_client_data["metadata"]["name"]
```

Comment:
Why is this changed? Could we deploy storage client with a different name?

Reply:
we need to choose where to get name for storage-client. Ideally we should have one source (template or constants). This is visible especially now, when we had to change it to "ocs-storagecluster" - w/a for upgrade issue. I had to find multiple sources for that name (bad initial design). 
Considering that the template is more hidden way I thought better to have a constant name.